R/inc_prop.R

```diff
@@ -37,7 +37,7 @@ inc_prop <- function(data,
   assert_true(tau > 0)
 
   n <- nrow(data)
-  ae <- nrow(data["type_of_event" == 1 & "time_to_event" <= tau]) / n
+  ae <- nrow(data[data$type_of_event == 1 & data$time_to_event <= tau, ]) / n
```

Comment:
actually this could be simplified to:
```suggestion
  ae <- sum(data$type_of_event == 1 & data$time_to_event <= tau) / n
```